### PR TITLE
Release notes/10.11.+

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-11-0/announcements-10-11-0.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+##### Usage of special characters for category and key for TenantOptionCollection API has been disallowed
+
+As announced with [release 10.9.0](/releasenotes/release-10-9-0/announcements-10-9-0/), creating tenant options with special characters in the category or key has been disabled to prevent the following issues:
+
+- The TenantOptionCollection API returns 400 instead of collection results due to the usage of special characters in category and key.
+- After creating a new tenant option with special characters in key or category, the generated self link points to a non-existing option.
+- When special characters are used the self link is broken and the user is unable to get or delete such an option via REST.
+
+For reference, we disable all HTTP-encoded and control characters (like \u0000). The full list of HTTP-encoded characters equals the one here: https://secure.n-able.com/webhelp/NC_9-1-0_SO_en/Content/SA_docs/API_Level_Integration/API_Integration_URLEncoding.html).
```
Comment (MWindel) [inline content/release-10-11-0/announcements-10-11-0.md:29]:
```suggestion
For reference, we disable all HTTP-encoded and control characters (like \u0000). The full list of HTTP-encoded characters equals the one here: https://secure.n-able.com/webhelp/NC_9-1-0_SO_en/Content/SA_docs/API_Level_Integration/API_Integration_URLEncoding.html.
```

File: content/release-10-11-0/announcements-10-11-0.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+### Security changes
+
+#### Planned
+
+##### Removal of Basic Auth browser-based authentication
+
+With the 10.5 release a new token-based mechanism for browser-based authentication was introduced (O-Auth Internal) in order to tighten the security of the Cumulocity IoT platform.
+
+With the 10.12 release, the O-Auth Internal authentication will be enabled by default for all tenants. With the 10.13 release the Basic Authentication option will be removed for browser-based applications and all applications  will be forced to use the token-based authentication mechanism O-Auth Internal. Note, that Basic Authentication will still be available for devices connecting to the Cumulocity IoT platform.
```
Comment (rssag) [inline content/release-10-11-0/announcements-10-11-0.md:39]:
```suggestion
With the 10.12 release, the O-Auth Internal authentication will be enabled by default for all tenants. With the 10.13 release the Basic Authentication option will be removed for browser-based applications and all applications will be forced to use the token-based authentication mechanism O-Auth Internal. Note that Basic Authentication will still be available for devices connecting to the Cumulocity IoT platform.
```

File: content/release-10-11-0/platform-10-11-0-bundle/10-11-0-4.md [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+SMS microservice</td>
+<td> The SMS-gateway microservice has been updated to use Telefonica Kite API v9. </td>
+<td>
+MTM-40883</td>
+</tr>
+
+<tr>
+<td>
+Two-factor authentication</td>
+<td > Cumulocity IoT will now provide the issuer name to distinguish between different  instances, tenants and other OTC stored in authenticator applications like Google Authenticator or Microsoft Authenticator. </td>
```
Comment (rssag) [inline content/release-10-11-0/platform-10-11-0-bundle/10-11-0-4.md]:
```suggestion
<td > Cumulocity IoT will now provide the issuer name to distinguish between different instances, tenants and other OTC stored in authenticator applications like Google Authenticator or Microsoft Authenticator. </td>
```

File: content/release-10-11-0/platform-10-11-0-bundle/10-11-0-4.md [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+Administration</td>
+<td > An issue with asynchronous tenant deletion has been resolved. Previously, when the platform failed to delete one tenant (for example due to database issues), then it stopped processing the deletion of a task. The fix contains handling errors, which now enable the deletion of other tenants to continue. Additionally, after a tenant deletion task failed for tenants, it retries the operation for up to 10 times. If it fails 10 times, then the affected tenant is suspended and an alarm is raised on the Management tenant.</td>
+<td>
+MTM-40638</td>
+</tr>
+
+<tr>
+<td>
+Administration</td>
+<td> Fixed a bug that was preventing administrators, logged in using SSO, to change local users email addresses.</td>
```
Comment (rssag) [inline content/release-10-11-0/platform-10-11-0-bundle/10-11-0-4.md:175]: i'd rather add a "when they were logged in using SSO" at the end

File: content/release-10-11-0/platform-10-11-0-bundle/10-11-0-4.md [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+Administration</td>
+<td> The <b>New tenant</b> window had some overlapping style issues on the <b>Storage limit per device</b> field, which resulted in unreadable error messages. These issues have been fixed.</td>
+<td>
+MTM-41498</td>
+</tr>
+
+<tr>
+<td>
+Billing</td>
+<td > The tracker-agent now uses microservice subscription. The registration  of tenants tracker-agent-{tenantId} is no longer required. A subscription is now required for new tenants using the tracker-agent. Tenants that have already been using tracker-agent will be subscribed automatically.</td>
```
Comment (rssag) [inline content/release-10-11-0/platform-10-11-0-bundle/10-11-0-4.md]:
```suggestion
<td > The tracker-agent now uses microservice subscription. The registration of tenants tracker-agent-{tenantId} is no longer required. A subscription is now required for new tenants using the tracker-agent. Tenants that have already been using tracker-agent will be subscribed automatically.</td>
```

File: content/release-10-11-0/whatsnew-10-11-0.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+At this time the Notifications 2.0 API is only available to tenants in the eu-latest.cumulocity.com public cloud environment. It can also be made available by request in private cloud environments. Contact [product support](/releasenotes/about/contacting-support/) to inquire about using Notifications 2.0 in your Cumulocity IoT environment.
+
+For details on the capabilities and usage of Notifications 2.0, see [Notifications 2.0](https://cumulocity.com/guides/reference/notifications/) in the *Reference guide* and the Cumulocity IoT [API documentation](https://cumulocity.com/api). The [Notifications 2.0 example microservice](https://github.com/SoftwareAG/cumulocity-examples/tree/develop/hello-world-notification-microservice) may also be a useful guide for users developing applications that will use the new API.
```
Comment (rssag) [inline content/release-10-11-0/whatsnew-10-11-0.md:44]: https://cumulocity.com/api/10.11.0/#tag/Notification-2.0-API
there's this already in 10.11.0 (which is not visible via landing page yet)

File: content/release-10-11-0/whatsnew-10-11-0.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+#### New API for timestamp
+
+In order to facilitate gathering alarm and event data across a period of date and time, two new query parameters are now available in the REST API. Measurement data is not in scope.  
+
+Existing queries do not need to be changed, they will behave in exactly the same manner as prior to release 10.10.  For date and time bounded queries the new parameters available are:
```
Comment (rssag) [inline content/release-10-11-0/whatsnew-10-11-0.md]:
```suggestion
Existing queries do not need to be changed, they will behave in exactly the same manner as prior to release 10.10. For date and time bounded queries the new parameters available are:
```

**Review by rssag — CHANGES_REQUESTED** [review]

File: content/release-10-11-0/announcements-10-11-0.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+The deprecated HOOK_COMPONENT has been removed in favor of HOOK_COMPONENTS.
+
+This change only affects you, if you or your development team use the Web SDK to extend Cumulocity IoT UI applications or to build your own web applications. If you update an application including HOOK_COMPONENT, make sure to use HOOK_COMPONENTS instead.
+
+
+### Other changes
+
+#### Planned
+
+##### Removal of cep microservice
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-10-11-0/announcements-10-11-0.md:118]: Yes, this is the name of the microservice.

File: content/release-10-11-0/devices-10-11-0-bundle/10-11-0-4.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+LWM2M</td>
+<td> The attribute <code>C8Y.lwm2m.enableBootstrapPSKFragmentFallback</code> has been added to the agent's default configuration. If enabled, the LWM2M agent will find the bootstrap PSK ID from the fragments if it cannot be found from the external IDs.</td>
+<td>
+MTM-40632</td>
+</tr>
+
+<tr>
+<td>
+OPC UA</td>
+<td> The processing mode set in the OPC UA device protocol is now also applied for the mappings created due to historic read operations with TAG tag type.</td>
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-10-11-0/devices-10-11-0-bundle/10-11-0-4.md:174]: Done

File: content/release-10-11-0/machine-learning-10-11-0-bundle/10_11_0.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+* Upgrades:
+  * Machine Learning web application upgraded from Angular version 8 to version 11.
+  * Nyoka and Onnx microservices upgraded from Python version 3.6 to version 3.7.11.
+  * Onnx microservice upgraded from onnx-runtime version 1.3.0 to version 1.5.2.
+  * Zementis microservice upgraded to use Red Hat UBI8 as it's base image.
+
+- Bug fixes:
+  - The HTTP response code for `{{url}}/service/zementis/train/timeseries` and `{{url}}/service/zementis/train/clustering` APIs updated from `200 - OK` to `202 - Accepted`.
+  - JSON parser now throws a consistent `400 - Bad Request` when an incorrect JSON is passed by the user. Earlier it was throwing `500 - Internal Server Error` in some cases.
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-10-11-0/machine-learning-10-11-0-bundle/10_11_0.md:26]: Done

File: content/release-10-11-0/platform-10-11-0-bundle/10-11-0-4.md (hunk 4 above)
Comment (BeateRixen) [reply content/release-10-11-0/platform-10-11-0-bundle/10-11-0-4.md:175]: Done

**Review by rssag — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**